py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2026,3 +2026,24 @@ def aten_ops_addmm(
         beta=kwargs.get("beta", 1),
         alpha=kwargs.get("alpha", 1),
     )
+
+
+@dynamo_tensorrt_converter(torch.nn.functional.interpolate)  # type: ignore[misc]
+def interpolate(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.interpolate.interpolate(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        input=args[0],
+        out_shape=kwargs.get("size", 1),
+        scale_factor=kwargs.get("scale_factor", 1),
+        interpolate_mode=kwargs.get("mode", 1),
+        align_corners=kwargs.get("align_corners", 1),
+    )
```

Comment:
The decorator's `key` shoud exactly match what the schema has, which typically is something like `torch.ops.aten.xxxxx`. And the type of arguments should also exactly match with the schema. After the corresponding changes, consider using `args_bounds_check(args, ...)` that takes arguments out of `args` instead of `kwargs`.

Reply:
from my understanding, when we have some function schema as: 
`
call_function[target=torch.ops.aten.upsample_bilinear2d.vec](args = (%input_1,), kwargs = {output_size: (4, 4), align_corners: True, scale_factors: None})
`
we should use `kwargs` since arguments like `output_size` are in kwargs instead, right?

Reply:
The reason why `output_size` and so on appeared in `kwargs` is that you passed in arguments in tests by: 
```
return torch.ops.aten.upsample_nearest2d.vec(
    input, output_size=output_shape, scale_factors=None
)
```
If you remove `=`, as you can see we don't use `=` in other tests, they should be in `args`. 

For the underlying reason, let's go back to the schema:
```
func: upsample_bilinear2d.vec(Tensor input, SymInt[]? output_size, bool align_corners, float[]? scale_factors) -> Tensor
```
They should belong to `args`, instead of `kwargs`. 
Only those arguments following `*` are considered as `kwargs`. For example,
```
replication_pad3d.out(Tensor self, SymInt[6] padding, *, Tensor(a!) out) -> Tensor(a!)
```
Based on the schema above, `self` and `padding` should be considered as `args` while `out` should be `kwargs`. The use of `*` in the function signature enforces this syntax.

I encountered this problem before as well, @gs-olive told me **"The current tracer being used by default in testing is a very barebones tracer, so it knows nothing about `aten` ops specifically. It seems that if an argument is specified with `=`, it will consider that a `kwarg`"**

Reply:
cool, thanks!